### PR TITLE
1.1.1 - arduino runtime

### PR DESCRIPTION
**Pull request by gbr1**
Added arduino runtime dependency

### CODE REVIEW

**Review by ubidefeo — APPROVED** [review]
LGTM

File: package.json [hunk 1]
```diff
@@ -10,7 +10,8 @@
         ["arduino_alvik/stm32_flash.py", "github:arduino/arduino-alvik-mpy/arduino_alvik/stm32_flash.py"]
     ],
     "deps": [
-        ["github:arduino/ucPack-mpy", "0.1.6"]
+        ["github:arduino/ucPack-mpy", "0.1.6"],
+        ["github:arduino/arduino-runtime-mpy"]
```
Comment (ubidefeo) [inline package.json:14]: add `, "latest"` to `arduino-runtime-mpy`

**Review by ubidefeo — CHANGES_REQUESTED** [review]

**Comment by gbr1** [comment]
@ubidefeo unfortunately we need the exact tag release

**Review by ubidefeo — APPROVED** [review]
tested with `mpremote mip install` and `mip.install()`

**Pull request merged**